.github/pull_request_template.md

```diff
@@ -9,25 +12,25 @@ Before you create this PR confirm that it meets all requirements listed below by
 - [ ] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
 - [ ] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors and are passing (See [Contributor Guide]).
 - [ ] My PR does not decrease the code coverage, or I have __a very special case__ and explained on the PR description why this PR decreases the coverage.
```

Comment:
```suggestion
- [ ] My PR does not decrease the code coverage.
```
(if anyone wants to decrease coverage, they can keep this unchecked and then explain in the PR).